v2/tools/generator/internal/astmodel/internal_type_name.go

```diff
@@ -186,6 +186,10 @@ func (tn InternalTypeName) IsStatus() bool {
 
 // IsARMType returns true if the TypeName identifies an ARM specific type, false otherwise.
 func (tn InternalTypeName) IsARMType() bool {
+	if IsARMPackageReference(tn.InternalPackageReference()) {
+		return true
+	}
+
 	return strings.HasSuffix(tn.Name(), ARMSuffix)
```

Comment:
Can this second bit be removed if we migrate all types to the new format?

Reply:
Once we've moved every group, I'll remove both this and the deny list.